core/src/main/java/org/apache/iceberg/PartitionStatsHandler.java

```diff
@@ -89,11 +89,16 @@ private PartitionStatsHandler() {}
       NestedField.optional(11, "last_updated_at", LongType.get());
   public static final NestedField LAST_UPDATED_SNAPSHOT_ID =
       NestedField.optional(12, "last_updated_snapshot_id", LongType.get());
+  public static final NestedField DV_COUNT =
+      NestedField.required(13, "dv_count", IntegerType.get());
```

Comment:
schema id and field name as per spec
https://github.com/apache/iceberg/pull/12098